audio/FAudio/Portfile

```diff
@@ -18,28 +18,18 @@ long_description        an XAudio reimplementation that focuses solely on develo
 
 depends_lib-append      port:libsdl2
 
-checksums               rmd160  a735305895754d503a71c03a71225513c0d55ddc \
-                        sha256  5599d4351b716e1dc8716b3ddd5855fddbf7a48f47558e04648a8a731cb39637 \
-                        size    1134780
+checksums               rmd160  dcdea2f9212da074dbae6485c250d78b3fa7d0b3 \
+                        sha256  d80a9598ec9e94652e8b7f5cf503b7c9792c8f905e63c9d93aa5d03a8efa2cf5 \
+                        size    1109448
 
 # remove set deployment target and hard-coded RPATH setting
 patchfiles              patch-faudio-remove-deployment-target.diff
 
-configure.args          -DGSTREAMER=OFF \
```

Comment:
Is this option gone from FAudio now or did we just remove it from the port?
If it's not gone, maybe we can explicitly set it to ON or OFF (it's not exactly clear to me from the logic below what it should be).

Reply:
This was removed from FAudio after v22.01, the only real consumer of FAudio + gstreamer was wine.

Wine now imported the FAudio sources and builds as PE format where access to other codecs is handled by other wine components.